src/features/map/ui/MapMarkers.tsx

```diff
@@ -0,0 +1,97 @@
+import { User, Pin, MultiplePin, Cluster } from "@/entities/map/ui";
+
+/*---------- default mode 일 때에만 사용되는 마커입니다. ---------- */
+
+export const UserMarker = () => {
+  const position = { lat: 37.5665, lng: 126.978 };
+
+  return <User position={position} />;
+};
+
+export const PinMarker = () => {
+  // TODO API 요청으로 가져오기
+  const markersInfo = [
+    {
+      alt: "test",
+      position: { lat: 37.56651, lng: 126.977 },
+      imageUrl: "/public/default-image.png",
+    },
+    {
+      alt: "test",
+      position: { lat: 37.56652, lng: 126.975 },
+      imageUrl: "/public/default-image.png",
+    },
+    {
+      alt: "test",
+      position: { lat: 37.56653, lng: 126.973 },
+      imageUrl: "/public/default-image.png",
+    },
+  ];
+
+  return markersInfo.map((markerInfo, idx) => (
+    <Pin {...markerInfo} key={idx} />
+  ));
+};
+
+export const MultiplePinMarker = () => {
+  // TODO API 요청으로 가져오기 ? 혹은 가져온 markersInfo를 이용하여 클러스터링 하기
+  const multiMarkerInfo = [
+    {
+      position: { lat: 37.5664, lng: 126.974 },
+      imageUrl: "/public/default-image.png",
+      alt: "test",
+      markerCount: 2,
+    },
+    {
+      position: { lat: 37.5663, lng: 126.972 },
+      imageUrl: "/public/default-image.png",
+      alt: "test",
+      markerCount: 500,
+    },
+  ];
+
+  return multiMarkerInfo.map((markerInfo, idx) => (
+    <MultiplePin {...markerInfo} key={idx} />
+  ));
+};
+
+export const ClusterMarker = () => {
+  // TODO API 요청으로 가져오기 ? 혹은 가져온 markersInfo를 이용하여 클러스터링 하기
+  const clusterInfo = [
+    {
+      position: { lat: 37.5662, lng: 126.97 },
+      markerCount: 16,
+    },
+    {
+      position: { lat: 37.5661, lng: 126.968 },
+      markerCount: 32,
+    },
+  ];
+
+  return clusterInfo.map((cluster, idx) => <Cluster {...cluster} key={idx} />);
```

Comment:
마커 관련 컴포넌트들의 JSX에서 최상단 태그가 없어요 👀 

```suggestion
return (<>
  {clusterInfo.map((cluster, idx) => <Cluster {...cluster} key={idx} />)}
</>)
```

그리고, 나중에 idx 대신 서버에서 주는 id 값으로 처리하실 예정이신가요?!

Reply:
```tsx
export const ClusterMarker = () => {
  // TODO API 요청으로 가져오기 ? 혹은 가져온 markersInfo를 이용하여 클러스터링 하기
  const clusterInfo = [
    {
      position: { lat: 37.5662, lng: 126.97 },
      markerCount: 16,
    },
    {
      position: { lat: 37.5661, lng: 126.968 },
      markerCount: 32,
    },
  ];

  return (
    <>
      {clusterInfo.map((cluster, idx) => (
        <Cluster {...cluster} key={idx} />
      ))}
    </>
  );
};
```

다음과 같이 수정했습니다 

클러스터들에서 새로운 상태가 추가되거나 제거되는 경우가 없기에 인덱스를 키 값으로 쓰는 것이 부적절해 보이진 않는 거 같습니다.

다만 인덱스를 키 값으로 쓰지 말라는 것이 거의 정론이기에 해당 부분이 안좋게 보인다면 변경 할 수도 있숩니다